.changeset/techdocs-and-then-you-cry.md

```diff
@@ -0,0 +1,22 @@
+---
+'@backstage/plugin-techdocs': minor
+---
+
+This plugin now exports two TechDocs addons which expose "core" TechDocs functionality for use within a TechDocs addon-enabled configuration. If you wish to use TechDocs addons, you can get the same search and state/build log indication experiences by adding the following two addons to your `<TechDocsAddons>` registry:
```

Comment:
> This plugin now exports two TechDocs addons which expose "core" TechDocs functionality 

Interesting, I was expecting all addons (that is contributed as a part of the "core" addons) to be exported from the techdocs-addons plugin and the techdocs plugin to just export components and apis that is crucial for the TechDocs experience. Would love to hear your thinking around it though.

Reply:
Indeed!  With the refactor of the framework, this is no longer necessary, so it's gone.